types/fake-response/index.d.ts

```diff
@@ -0,0 +1,214 @@
+ */
+export class FakeResponse {
+    constructor(db?: UserDB, config?: Config, globals?: Globals, injectors?: Injectors[]);
+    /**
+     * This function creates express app, starts the server loads the resources and creates default routes.
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse();
+     * fakeResponse.launchServer() // if not provided anything in the constructor runs the sample db
+     * @link https://r35007.github.io/Fake-Response/#launchserver - For further info pls visit this ReadMe
+     */
+    launchServer: () => Promise<{
+        app: express.Application;
+        server: Server;
+        results: RouteResult[];
+        db: Db[];
+        config: Config;
+        globals: Globals;
+        fullDbData: object;
+    }>;
+    /**
+     * This function creates express app with default middlewares.
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const app = fakeResponse.createExpressApp(); // creates and returns the express app.
+     * @link https://r35007.github.io/Fake-Response/#createexpressapp - For further info pls visit this ReadMe
+     */
+    createExpressApp: () => express.Application;
+    /**
+     * This function starts the express server.
+     * Please make sure you create the express app before starting the server
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const app = fakeResponse.createExpressApp();
+     * fakeResponse.startServer(3000); // the port is an optional param
+     * @link https://github.com/R35007/Fake-Response#startserver - For further info pls visit this ReadMe
+     */
+    startServer: (port?: number) => Promise<Server>;
+    /**
+     * This function stops the express server
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const isStopped = fakeResponse.stopServer(); // make sure the server is already started
+     * @link https://github.com/R35007/Fake-Response#stopserver - For further info pls visit this ReadMe
+     */
+    stopServer: () => Promise<boolean>;
+    /**
+     * This function helps to generate a route before starting the server
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     * const result = fakeResponse.loadResources();
+     * @link https://github.com/R35007/Fake-Response#loadresources - For further info pls visit this ReadMe
+     */
+    loadResources: () => Promise<RouteResult[]>;
+    /**
+     * This function helps to create a route explicitly
+     * @example
+     * const {FakeResponse} = require("fake-response");
+     * const fakeResponse = new FakeResponse(db);
+     *
+     *const newResponse = {
+          value : "New response"
+     }
```

Comment:
```suggestion
     * const newResponse = {
     *     value : "New response"
     * }
```
again, this can be mitigated before submitting changes with `npm run lint fake-response`